locale/ru_RU/currencies.xml

```diff
@@ -8,177 +8,170 @@
   * Copyright (c) 2000-2016 John Willinsky
   * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
   *
-  * Localized list of currencies.
-  *
+  * Currencies data XML file. Based on ISO 4217.
```

Comment:
I've translated the file from en_US locale and checked it with some Russian sources.